cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: glbc
+  namespace: kube-system
+  labels:
+    k8s-app: glbc
+    version: v0.5
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "GLBC"
+spec:
+  # There should never be more than 1 controller alive simultaneously.
+  replicas: 1
+  selector:
+    k8s-app: glbc
+    version: v0.5
+  template:
+    metadata:
+      labels:
+        k8s-app: glbc
+        version: v0.5
+        name: glbc
+        kubernetes.io/cluster-service: "true"
+    spec:
+      terminationGracePeriodSeconds: 600
+      containers:
+      - image: gcr.io/google_containers/defaultbackend:1.0
+        name: default-backend
+        # Any image is permissable as long as:
+        # 1. It serves a 404 page at /
+        # 2. It serves 200 on a /healthz endpoint
+        image: gcr.io/google_containers/defaultbackend:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+        ports:
+        - containerPort: 8080
+        resources:
+          limits:
+            cpu: 10m
+            memory: 20Mi
+          requests:
+            cpu: 10m
+            memory: 20Mi
+      - image: gcr.io/google_containers/glbc:0.5
```

Comment:
played around with it a bit and ended up with l7-lb-controller for the rc/container (figured the right controller would be on the right platform since this is an addon), and default-http-backend for the service, but leaving app: glbc as is.